src/Build.OM.UnitTests/Definition/ProjectItem_Tests.cs

```diff
@@ -2713,5 +2715,40 @@ private static ProjectItem GetOneItem(string content)
             Assert.Equal(1, items.Count);
             return items[0];
         }
+
+        /// <summary>
+        /// Item metadata "Filename" should not depends on platform specific slashes.
+        /// </summary>
+        [Fact]
+        public void FileNameMetadataEvaluationShouldNotDependsFromPlatformSpecificSlashes()
```

Comment:
Rather than manually handling files and the project collection, you could use Microsoft.Build.UnitTests.TestEnvironment.CreateTestProjectWithFiles to handle everything for you.